packages/upload-client/README.md

```diff
@@ -15,27 +15,33 @@ npm install @web3-storage/upload-client
 
 ### Create an Agent
 
-An Agent provides an `issuer` (a key linked to your account) and `proofs` to show your `issuer` has been delegated the capabilities to store data and register uploads.
+An Agent provides:
+
+1. The key pair used to call the service and sign the payload (the `issuer`).
+2. A decentralized identifier (DID) of the "space" where data should be uploaded (the `with`).
+3. Proof showing your `issuer` has been delegated capabilities to store data and register uploads to the "space" (`proofs`).
 
 ```js
 import { Agent } from '@web3-storage/access'
-import { add as storeAdd } from '@web3-storage/access/capabilities/store'
-import { add as uploadAdd } from '@web3-storage/access/capabilities/upload'
+import { store } from '@web3-storage/access/capabilities/store'
+import { upload } from '@web3-storage/access/capabilities/upload'
 
-const agent = await Agent.create({ store })
+const agent = await Agent.create()
 
-// Note: you need to create and register an account 1st time:
-// await agent.createAccount('you@youremail.com')
+// Note: you need to create and register a space 1st time:
+// await agent.createSpace()
+// await agent.registerSpace('you@youremail.com')
```

Comment:
This is TBC...skating to where the puck will be based on https://github.com/web3-storage/w3protocol/pull/177